spec/feature/hearings/virtual_hearings/hearing_details_spec.rb

```diff
@@ -121,576 +120,550 @@ def ensure_link_present(link, disable)
-      expect(events.sent_to_appellant.count).to eq 1
-      expect(events.where(email_address: central_virtual_hearing.representative_email).count).to eq 1
-      expect(events.where(recipient_role: "representative").count).to eq 1
+        expect(page).to have_field("appellantTz")
 
-      # Check the Email Notification History
-      check_email_event_table(central_hearing, 2)
-    end
-  end
+        events = SentHearingEmailEvent.where(hearing_id: hearing.id)
+        expect(events.count).to eq 2
+        expect(events.where(sent_by_id: current_user.id).count).to eq 2
+        expect(events.where(email_type: "confirmation").count).to eq 2
+        expect(events.where(email_address: virtual_hearing.appellant_email).count).to eq 1
+        expect(events.sent_to_appellant.count).to eq 1
+        expect(events.where(email_address: fill_in_rep_email).count).to eq 1
+        expect(events.where(recipient_role: "representative").count).to eq 1
 
-  context "Updating either Appellant and POA/Representative email address and timezone" do
-    let!(:central_virtual_hearing) do
-      create(:virtual_hearing,
-             :all_emails_sent,
-             :previously_central,
-             status: :active,
-             hearing: central_hearing)
+        # Check the Email Notification History
+        check_email_event_table(hearing, 2)
+      end
     end
+  end
 
-    scenario "Sends the confirmation email" do
-      visit "hearings/" + central_hearing.external_id.to_s + "/details"
-
-      fill_in "POA/Representative Email", with: fill_in_rep_email
-      click_dropdown(name: "appellantTz", index: 1)
-      click_button("Save")
-
-      expect(page).to have_content(COPY::VIRTUAL_HEARING_MODAL_UPDATE_GENERIC_TITLE)
-      expect(page).to have_content(COPY::VIRTUAL_HEARING_UPDATE_EMAIL_BUTTON)
-      click_button(COPY::VIRTUAL_HEARING_UPDATE_EMAIL_BUTTON)
-
-      visit "hearings/" + central_hearing.external_id.to_s + "/details"
+  context "Initally a Video hearing" do
+    let!(:hearing) { create(:hearing, :with_tasks, regional_office: "RO13") }
 
-      expect(page).to have_field("appellantTz")
+    include_context "shared behaviors"
+  end
 
-      events = SentHearingEmailEvent.where(hearing_id: central_hearing.id)
-      expect(events.count).to eq 2
-      expect(events.where(sent_by_id: current_user.id).count).to eq 2
-      expect(events.where(email_type: "confirmation").count).to eq 2
-      expect(events.where(email_address: central_virtual_hearing.appellant_email).count).to eq 1
-      expect(events.sent_to_appellant.count).to eq 1
-      expect(events.where(email_address: fill_in_rep_email).count).to eq 1
-      expect(events.where(recipient_role: "representative").count).to eq 1
+  context "Initally a Central hearing " do
+    let!(:hearing) { create(:hearing, :with_tasks) }
 
-      # Check the Email Notification History
-      check_email_event_table(central_hearing, 2)
-    end
+    include_context "shared behaviors"
```

Comment:
```suggestion
    include_examples "shared behaviors"
```
see note about `shared_examples`